### PR TITLE
networking: Use radio buttons for MTU dialog

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by andreasn** [comment]
I get this massive input field for some reason. It's also quite long, making me believe I need to input a very large number.

screenshot from 2016-07-11 16-11-38

As discussed on IRC, probably makes sense to drop "Bytes" after the custom input.
Apart from that, looks good. Thanks for the quick fix!

**Comment by stefwalter** [comment]
@mvollmer Unfortunately that doesn't look like the designs. A bit more work is needed to bring it in line:

mockup2

**Comment by mvollmer** [comment]
> Unfortunately that doesn't look like the designs.

Yes, there isn't any real styling yet.

**Comment by andreasn** [comment]
Marius pointed out on IRC that the label MTU is unnecessary, so I fixed that (and removed 'bytes')

mtu-size-dialog-v3

**Comment by mvollmer** [comment]
Looks like this now.  Changing the color of the radio knob doesn't seem to be straightforward.

shot

**Comment by andreasn** [comment]
I think the knob color is good like this.

**Comment by andreasn** [comment]
We could achieve the proxy-controls with a table somewhat like this:
http://pastebin.com/wNuupdWX

File: pkg/networkmanager/index.html [hunk 1]
```diff
@@ -136,17 +136,21 @@
...
+               type="radio" name="ethernet-mtu-mode"></input>
+        <span translatable="yes">Automatic</span>
+      </label>
+    </div>
+    <div>
+      <label>
+        <input id="network-ethernet-settings-mtu-custom"  {{#mtu}}checked="checked"{{/mtu}}
+               type="radio" name="ethernet-mtu-mode"></input>
+        <span translatable="yes">Set to</span>
+        <input id="network-ethernet-settings-mtu-input" type="text" value="{{mtu}}"></input>
```
Comment (andreasn) [inline pkg/networkmanager/index.html]: This needs a form-control class.
This unfortunately makes the input end up on it's own row, so need some override css for doing display: inline;

Comment (andreasn) [inline pkg/networkmanager/index.html]: Fixed in e4b60e9f2436dcca3f6eb78adcfd4c118719cd6b

**Comment by dperpeet** [comment]
needs a rebase

**Comment by mvollmer** [comment]
> needs a rebase

Done.

**Comment by stefwalter** [comment]
Failure seems related:

```
# testBasic (check_networking.TestNetworking)
#
52:54:00:9e:00:F2 -> ens6
52:54:00:9e:00:F2 -> ens6
Page error: tr:contains('MTU') a is not visible
phantomjs://code/phantom-lib.js 108 ph_click
 1 
 1 
Wrote TestNetworking-testBasic-FAIL.png
```

**Comment by mvollmer** [comment]
> ```
> Page error: tr:contains('MTU') a is not visible
> ```

Good catch, testsuite.  The MTU rows was still putting a <div> into a <a>.

**Pull request closed without merging**